app-backend/app/src/main/scala/scene/Routes.scala

```diff
@@ -39,9 +40,8 @@ trait SceneRoutes extends Authentication
 
   def listScenes: Route = authenticate { user =>
     (withPagination & sceneQueryParameters) { (page, sceneParams) =>
-      complete {
-        Scenes.listScenes(page, sceneParams, user)
-      }
+      RfPagination.validatePage(page)
```

Comment:
I think this is a good start, I think it probably should be a new directive that returns a page and replaces the `withPagination` directive provided by the library we are using.

I think it would look something like [this](https://github.com/lonelyplanet/akka-http-extensions/blob/master/src/main/scala/com/lonelyplanet/akka/http/extensions/PaginationDirectives.scala#L75), but with an additional case where the offset is invalid (e.g. -1) and it's filled in with 0.

I can pair on this tomorrow to help get started. I think this is pretty close.